unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/ChannelsUtils.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.Chat
+{
+    public class ChannelsUtils : IChannelsUtils
+    {
+        private const string FEATURE_FLAG_FOR_USERS_ALLOWED_TO_CREATE_CHANNELS = "users_allowed_to_create_channels";
+        private const string VARIANT_FOR_USERS_ALLOWED_TO_CREATE_CHANNELS = "allowedUsers";
+
+        private BaseVariable<FeatureFlag> featureFlags => DataStore.i.featureFlags.flags;
+
+        private readonly DataStore dataStore;
+        private readonly IUserProfileBridge userProfileBridge;
+
+        public ChannelsUtils(DataStore dataStore, IUserProfileBridge userProfileBridge)
+        {
+            this.dataStore = dataStore;
+            this.userProfileBridge = userProfileBridge;
+        }
+
+        public bool IsAllowedToCreateChannels()
+        {
+            if (!featureFlags.Get().IsFeatureEnabled(FEATURE_FLAG_FOR_USERS_ALLOWED_TO_CREATE_CHANNELS))
+                return false;
+
+            FeatureFlagVariantPayload usersAllowedToCreateChannelsPayload = featureFlags
+                .Get()
+                .GetFeatureFlagVariantPayload($"{FEATURE_FLAG_FOR_USERS_ALLOWED_TO_CREATE_CHANNELS}:{VARIANT_FOR_USERS_ALLOWED_TO_CREATE_CHANNELS}");
+
+            if (usersAllowedToCreateChannelsPayload == null)
+                return false;
+
+            UsersAllowedToCreateChannelsVariantPayload allowedUsersData = JsonUtility.FromJson<UsersAllowedToCreateChannelsVariantPayload>(usersAllowedToCreateChannelsPayload.value);
+            UserProfile ownUserProfile = userProfileBridge.GetOwn();
+
+            switch (allowedUsersData.mode)
+            {
+                case AllowChannelsCreationMode.ALLOWLIST:
+                    List<string> allowedWallets = new List<string>();
+                    if (allowedUsersData.allowList != null)
```

Comment:
you can simplify: `return allowedUsersData.allowList.Any(userId => userId.ToLower() == ownUserProfile.userId.ToLower()`

Reply:
You're totally right! Done in:
- https://github.com/decentraland/unity-renderer/pull/3195/commits/4313a96e94053a2e0f5fbbe30ccbd7a8e5805825
- https://github.com/decentraland/unity-renderer/pull/3195/commits/aa5ebb8e7b7d050dec1fa3db3a70cfe55f959f8b